From d0c347811ffa1c865d16450e38e78f5182cf2e90 Mon Sep 17 00:00:00 2001
From: "dbatzle@dcbcyber.com" <dbatzle@dcbcyber.com>
Date: Tue, 25 Aug 2015 20:55:30 -0400
Subject: [PATCH 2/2] ARM Fix pandaboard es and a4 revision ID

board_name environment variable was not getting set correctly for Pandaboard A4 and ES

Signed-off-by: David Batzle <dbatzle@dcbcyber.com>
CC: Albert Aribaud <albert.u.boot@aribaud.net>; Tom Rini <trini@ti.com>; Peter Robinson <pbrobinson@gmail.com>
---
 board/ti/panda/panda.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/ti/panda/panda.c b/board/ti/panda/panda.c
index 783ba35..92e1b63 100644
--- a/board/ti/panda/panda.c
+++ b/board/ti/panda/panda.c
@@ -101,7 +101,7 @@ int get_board_revision(void)
 		board_id4 = gpio_get_value(PANDA_ES_BOARD_ID_4_GPIO);
 
 #ifdef CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
-		setenv("board_name", strcat(CONFIG_SYS_BOARD, "-es"));
+		setenv("board_name", "panda-es");
 #endif
 		board_id = ((board_id4 << 4) | (board_id3 << 3) |
 			(board_id2 << 2) | (board_id1 << 1) | (board_id0));
@@ -115,7 +115,7 @@ int get_board_revision(void)
 
 #ifdef CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
 		if ((board_id >= 0x3) && (processor_rev == OMAP4430_ES2_3))
-			setenv("board_name", strcat(CONFIG_SYS_BOARD, "-a4"));
+			setenv("board_name", "panda-a4");
 #endif
 	}
 
-- 
2.6.2

